.github/workflows/biome-check.yml

```
@@ -0,0 +1,41 @@
+name: Biome Check
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)